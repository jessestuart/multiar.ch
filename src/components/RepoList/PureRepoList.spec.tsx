import { DockerHubRepo } from 'docker-hub-utils'
import _ from 'lodash'
import React from 'react'
import renderer from 'react-test-renderer'

import PureRepoList from 'components/RepoList/PureRepoList'
import { getReposToArchitecturesMap } from 'utils/repos'

import fixtures from '../../../test/fixtures/DockerHubReposFixture'

const reposFixtures = (fixtures as unknown) as DockerHubRepo[]

describe('PureRepoList component.', () => {
  beforeAll(() => {
    jest.mock('gatsby')
    jest.mock('../Apollo')
  })

  test('Render PureRepoList component.', () => {
    const component = renderer.create(
      <PureRepoList
        repos={reposFixtures}
        repoToArchitecturesMap={getReposToArchitecturesMap(reposFixtures)}
      />,
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

  test('Render empty PureRepoList.', () => {
    const component = renderer.create(
      <PureRepoList
        repos={[]}
        repoToArchitecturesMap={getReposToArchitecturesMap([])}
      />,
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
