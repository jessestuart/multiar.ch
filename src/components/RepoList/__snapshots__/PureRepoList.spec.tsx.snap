// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PureRepoList component. Render PureRepoList component. 1`] = `
.c1 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  font-family: mono;
}

.c2 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  color: primary;
}

.c3 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
}

.c4 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  font-family: sans;
  color: textMuted;
}

.c0 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<div
  className="c0 center f4 justify-center pt4"
>
  <ol
    className="mw7-ns"
  >
    <li
      className="flex fw7 lh-copy"
    >
      <div
        className="c1 flex-nowrap flex-auto"
        fontFamily="mono"
      >
        Image
      </div>
      <div
        className="c1 flex justify-end"
        fontFamily="mono"
      >
        Pulls
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/owntracks"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            owntracks
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          OwnTracks. Multiarch. ðŸ‘Œ
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/caddy"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            caddy
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          The every(wo)man's reverse proxy, on all your boxes or boards. amd64, arm64, & armhf nightly builds.
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/tiller"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            tiller
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          Nightly multi-architecture (amd64, arm64, armv7) builds for the Kubernetes Helm tiller service.
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/renovate"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            renovate
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          All the dependencies, up-to-date, all the time. Nightly multiarch builds. (amd64 + arm64)
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/minio"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            minio
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          Minio server â€” supports arm (arm32v6, arm32v7, armhf) arm64 (aarch64), and amd64 architectures.
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/grafana"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            grafana
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          An OSS metrics dashboard and graph editor for Graphite, Elasticsearch, Prometheus and InfluxDB.
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/mc"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            mc
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          amd64, arm64, and armhf Docker images for Minio Client.
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/fluent-bit"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            fluent-bit
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          Multiarch (ARM, ARM64, AMD64) fluent-bit image. Nightly builds from master. Telemetry FTW.
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/unifi_exporter"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            unifi_exporter
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          Multi-arch (AMD, ARM, ARM64) image for scraping data from a Unifi controller.
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/velero"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            velero
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          Nightly multiarch (arm64, armhf, amd64) builds of Heptio Velero for Kubernetes.
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/ark"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            ark
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          Nightly multiarch (arm64, armhf, amd64) builds of Heptio's ark.
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/kube-local-volume-provisioner"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            kube-local-volume-provisioner
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          A local PV represents a local disk directly-attached to a Node. This reached GA in 1.14.
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/kube2iam"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            kube2iam
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          kube2iam provides different AWS IAM roles for pods running on Kubernetes.
 Nightly multiarch builds.
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/elasticsearch_exporter"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            elasticsearch_exporter
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/docker-hub-exporter"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            docker-hub-exporter
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/consul"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            consul
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/homemaker"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            homemaker
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/lmsensors_exporter"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            lmsensors_exporter
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/pihole_exporter"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            pihole_exporter
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/prometheus-operator"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            prometheus-operator
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/fluent-bit-builder"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            fluent-bit-builder
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/caddy-cloudflare"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            caddy-cloudflare
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          Caddy w/ Cloudflare plugin included. [NOTE: use jessestuart/caddy instead for nightly builds!] 
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/vagrant-build-base"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            vagrant-build-base
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/prometheus-config-reloader"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            prometheus-config-reloader
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/node_exporter"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            node_exporter
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/helm-exporter"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            helm-exporter
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/elasticsearch-kubernetes"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            elasticsearch-kubernetes
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/adguard-home"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            adguard-home
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/drone-server"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            drone-server
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/docker-registry"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            docker-registry
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/jenkins"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            jenkins
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/jesses.io"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            jesses.io
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/pihole-exporter"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            pihole-exporter
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={
        Object {
          "borderBottom": "1px rgba( 0, 0, 0, 0.1) solid",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/alpine-essentials"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            alpine-essentials
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
    <li
      className="flex lh-copy pt2 pb3"
      style={null}
    >
      <div
        className="css-vurnku"
      >
        <a
          className="c2 flex-nowrap inline-flex primary"
          color="primary"
          href="https://hub.docker.com/r/jessestuart/arm_exporter"
        >
          <div
            className="c1 fw5 pr2"
            fontFamily="mono"
          >
            arm_exporter
          </div>
        </a>
        <div
          className="c3 lh-title pv2"
        >
          
        </div>
        <div
          className="c4 flex flex-column f6 justify-start no-underline"
          color="textMuted"
          fontFamily="sans"
        >
          Updated 
          4 days ago
        </div>
      </div>
      <div
        className="c1 flex flex-auto justify-end pl4"
        fontFamily="mono"
      >
        <span />
      </div>
    </li>
  </ol>
</div>
`;

exports[`PureRepoList component. Render empty PureRepoList. 1`] = `null`;
